apps/mochi-web/components/Profile/PaymeLinksTable.tsx

```diff
@@ -46,12 +81,14 @@ export const PaymeLinksTable = () => {
         {
           header: 'Wen',
           accessorKey: 'created_at',
-          accessorFn: (row) => format(new Date(row.created_at ?? ''), 'Pp'),
+          accessorFn: (row) =>
+            formatDate(row.created_at ?? '', 'dd/MM/yyyy hh:mma'),
```

Comment:
Maybe it's safer to hide the data instead of showing the wrong info

```suggestion
            row.created_at ? formatDate(row.created_at, 'dd/MM/yyyy hh:mma') : null,
```